src/index.js

```diff
@@ -1,3 +1,11 @@
+// Global definitions
+// These need to be imported here to be placed in the distribution CSS file
+/* eslint-disable no-unused-vars */
+import themeStyles from './theme.scss';
+import foundationStyles from './foundation.scss';
+import helpersStyles from './helpers.scss';
+/* eslint-enable no-unused-vars */
+
```

Comment:
What kind of "distribution CSS file"? Current usage guidelines `/docs/getting-started/usage/#foundation-css` tell users to import these manually.

Is it related to https://github.com/react-ui-org/react-ui/issues/197? If so, we must update the docs.